src/ResourceManager/IotCentral/Commands.IotCentral/ChangeLog.md

```diff
@@ -19,3 +19,9 @@
 -->
 ## Current Release
 * Preview Release
+* Added Tabular Output Format
+* Added Subdomain Checking in New-AzureRmIotCentralApp
+
+## 2018.11.06 - Version 1.0.0
```

Comment:
Please only make edits to the ##Current release header

Reply:
@maddieclayton i updated I believe, am i doing that right now?